tests/collections/asr/test_asr_ctcencdec_model.py

```diff
@@ -89,10 +87,12 @@ def test_constructor(self):
             {'preprocessor': DictConfig(preprocessor), 'encoder': DictConfig(encoder), 'decoder': DictConfig(decoder)}
         )
         asr_model = EncDecCTCModel(cfg=modelConfig)
+        # WHAT IS THAT?!
         asr_model.train()
```

Comment:
... we can "train" without a scheduler or Optimizer? @okuchaiev

Reply:
@tkornuta-nvidia For now, maybe revert this line and remove your comment

Reply:
I honestly do not know what is happening here... moreover, IF there is training, then it is clearly not unit tests...

Reply:
This is just a call of  torch.nn.Module.train() method.

Reply:
eheheh, train()/eval()! right... ;)